openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/DataQualityTab.style.less

```
@@ -10,11 +10,37 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
+@import url('../../../styles/variables.less');
```

```suggestion
@import (reference) url('../../../styles/variables.less');
```

To avoid it's being injected multiple times